src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
In that case, we can remove it for now and bring it back if something is found that needs it. Main goal should be to focus on current/future updates. The versioning there is only an attempt to keep older versions working (and I'd be amazed if some of the older versions in this actually still work)